### ISSUE TITLE
Single Versus Multiple Individuals

### ISSUE THREAD

**Issue by Eystein-Thanisch** [cyocum/irish-gen#125]
Sometimes, one version of the genealogies will interpret a name as denoting one person and another version will take it as multiple people. For example, in their respective sections on Síl Birn, Rawl.B.502 has two brothers called "Daig" and "Linga", whereas LL has one person called "Daig Linguae". 

In such situations, is it acceptable to use `owl:sameAs` to link the multiple people in one version to the single person in the other version? On the one hand, this seems the best intepretation of the latter but, on the other, it could imply that the multiple people in the former are the same as each other, when the point is that they are different.

**Comment by cyocum** [comment]
Yes, I think I just made _ad hoc_ decisions which is bad.  I usually just made the seperated ones all `owl:sameAs` the combined one which makes the search work correctly if you have reasoning enabled (you will get some descendant of these individuals) but I am not sure if that is the right thing to do.
